### PR TITLE
Update spacemesh0.1.md

### PR DESCRIPTION
**Pull request by barakshani**

### CODE REVIEW

File: spacemesh0.1.md [hunk 1]
```diff
@@ -14,23 +14,23 @@ This guide walks you though and describes the features and the supported functio
 
 Here are some scenarios that are not yet supported in the Spacemesh 0.1 Testnet release:
```
Comment (barakshani) [inline spacemesh0.1.md]: maybe _features_ instead of _scenarios_?

File: spacemesh0.1.md [hunk 2]
```diff
@@ -14,23 +14,23 @@ This guide walks you though and describes the features and the supported functio
...
-1. Join the Testnet from a Node and a CLI wallet built directly from source code.
-2. Get Smesh coins from a tap.
+1. Join the Testnet from a Node and a CLI wallet built directly from source code. 
+2. Get Smesh coins from a tap account.
 3. Browse the Testnet using a block explorer and a dashboard.
 4. Run the App in Wallet-only mode without a Node.
 5. Run a node without producing blocks.
 
-We plan to introduce these roll out software to support these use cases in subsequent Spacemesh Testnet releases throughout 2020.
+We plan to introduce these roll out software to support these use cases in subsequent Spacemesh Testnet releases throughout 2020.
```
Comment (barakshani) [inline spacemesh0.1.md]: I don't understand this sentence

File: spacemesh0.1.md [hunk 3]
```diff
@@ -14,23 +14,23 @@ This guide walks you though and describes the features and the supported functio
...
-- Network id: 0.
-- POST init size per miner:	~20 minutes on a modern CPU.
-- Layer mining award: 50 SMCH.
-- Awards maturity: 4 layers (20 minutes).
-- Non-routable Nodes are not supported.
-- Layer duration: 5	minutes.
-- Number of layers in an epoch:	288.
-- Epoch duration: 24 hours.
-- Coin transfer transaction fee: 1-3 Smetoshis.
-- Time to first award: ~49 hours.
```
Comment (barakshani) [inline spacemesh0.1.md]: This page has two sections with bullet points. Either both or none should have a period in the end of each sentence. I removed all periods here.

File: spacemesh0.1.md [hunk 4]
```diff
@@ -14,23 +14,23 @@ This guide walks you though and describes the features and the supported functio
...
-We plan to introduce these roll out software to support these use cases in subsequent Spacemesh Testnet releases throughout 2020.
+We plan to introduce these roll out software to support these use cases in subsequent Spacemesh Testnet releases throughout 2020. 
 
 ## Spacemesh 0.1 Cheat Sheet
 
-- Network id: 0.
-- POST init size per miner:	~20 minutes on a modern CPU.
-- Layer mining award: 50 SMCH.
-- Awards maturity: 4 layers (20 minutes).
-- Non-routable Nodes are not supported.
```
Comment (barakshani) [inline spacemesh0.1.md]: This should not be here. Please move to the previous section of non-supported features.

File: spacemesh0.1.md [hunk 5]
```diff
@@ -14,23 +14,23 @@ This guide walks you though and describes the features and the supported functio
 
 Here are some scenarios that are not yet supported in the Spacemesh 0.1 Testnet release:
 
-1. Join the Testnet from a Node and a CLI wallet built directly from source code.
-2. Get Smesh coins from a tap.
+1. Join the Testnet from a Node and a CLI wallet built directly from source code.
```
Comment (barakshani) [inline spacemesh0.1.md]: Is it really not supported? What happens if I compile the Node and tries to connect to the netwrok?

File: spacemesh0.1.md [hunk 6]
```diff
@@ -14,23 +14,23 @@ This guide walks you though and describes the features and the supported functio
...
-- Layer duration: 5	minutes.
-- Number of layers in an epoch:	288.
-- Epoch duration: 24 hours.
-- Coin transfer transaction fee: 1-3 Smetoshis.
-- Time to first award: ~49 hours.
+- Network id: 0
+- POST init size per miner:	TBD (about 20 minutes on a modern CPU)
+- Layer mining award: 50 SMCH
+- Award maturity: After 4 fully conifermed layers (about 20 minutes)
+- Non-routable Nodes are not supported
```
Comment (barakshani) [inline spacemesh0.1.md]: This should not be here. Please move to the previous section of non-supported features.

File: spacemesh0.1.md [hunk 7]
```diff
@@ -14,23 +14,23 @@ This guide walks you though and describes the features and the supported functio
...
+- Layer duration: 5	minutes
+- Number of layers in an epoch:	288
+- Epoch duration: 24 hours
+- Coin transfer transaction fee: 1-3 Smetoshis
+- Time to first award: 24-72 hours
```
Comment (barakshani) [inline spacemesh0.1.md]: This page has two sections with bullet points. Either both or none should have a period in the end of each sentence. I removed all periods here.

File: spacemesh0.1.md (hunk 5 above)
Comment (avive) [reply spacemesh0.1.md]: I wish we'll be able to support this but I from my understanding more work is involved and in the last meet it was requested to delay it to 0.x. It is beyond just joining the network - this is about being able to see post, execute txs w the cli wallets and see awards in the cli wallet - in other words, a reasonable user experience.

File: main.md [hunk 8]
```diff
@@ -29,8 +29,8 @@ We aim to create a world in which any interested party can mine Spacemesh crypto
 ### Energy consumption due to PoW chains is skyrocketing.
 Spacemesh has improved upon PoW by creating a consensus protocol that consumes significantly less energy (i.e., `A Better Bitcoin`).
 
-### Lack of satisfactory smart contracts infrastructure for creation of fair communities and decentralized financial services.
-We plan to add a modern smart contracts virtual machine, called Spacemesh virtual machine (SVM), a new modern smart contracts language (Smesh) and dev tools designed for developers to build applications on the Spacemesh platform.
+### Lack of satisfactory smart contract infrastructure for creation of fair communities and decentralized financial services.
```
Comment (barakshani) [inline main.md]: What is a "fair community"?
More generally, the explanation below says nothing about the (potential) uses of smart contrasts. I believe that we should simply write "Lack of satisfactory smart contract infrastructure". What exactly the world will find fit to do with it is not one of our goals.

File: main.md [hunk 9]
```diff
@@ -64,19 +64,19 @@ Joining also helps you to prepare for participating in the Spacemesh Mainnet onc
...
 Just follow the instructions in this guide. Let's [get started](guide/install.md)!
 
 ## What do I need to join the Testnet?
-To run a full p2p node, mine and to get coin awards you need `an always-on desktop computer` running Windows 10, OS X or a supported Linux distribution with at least 150 GB of free disk space (required free space is tentative and subject to change) on one of your hard drives. For more information browse the [full system requirements](requirements.md).
+To run a full p2p node, mine and to get coin awards you need an `always-on desktop computer` running Windows 10, OS X or a supported Linux distribution with at least 150 GB of free disk space (required free space is tentative and subject to change) on one of your hard-drives. For more information browse the [full system requirements](requirements.md).
 
 ## Testnet Community Resources
 Got stuck? Need additional info? Got some cool ideas for the project?
 - Get answers and ask questions in our [Testnet Community Channel](https://discord.gg/Mf8T4Db) on Discord.
 - Get some Testnet Spacemesh Coins via our [Testnet Tap](https://discord.gg/Mf8T4Db) on Discord.
```
Comment (barakshani) [inline main.md]: This should be removed

File: main.md [hunk 10]
```diff
@@ -40,22 +40,22 @@ A mesh topology combined with a novel consensus PoST protocol that allows anyone
...
 
 ## Why should I join the Spacemesh Testnet?
-Joining the Testnet helps Spacemesh launch a high-quality Mainnet and the `Spacemesh Coin` (SMC) cryptocurrency. Spacemesh is a decentralized ledger designed to be formed from a large number of home desktop PC owners who run the Spacemesh software on their PCs. To test such a network we need a large number of block producers to join the Testnet in addition to the close Testnets which we have been running.
+Joining the Testnet helps Spacemesh launch a high-quality Mainnet and the `Spacemesh Coin` (SMC) cryptocurrency. Spacemesh is a decentralized ledger designed to be formed from a large number of home desktop PC owners who run the Spacemesh software on their PCs. To test such a network we need a large number of block producers to join the Testnet in addition to the close Testnets which we have been running.
```
Comment (barakshani) [inline main.md]: I don't see any value in stating that we have close testnets, and would remove this part of the sentence.

**Pull request closed without merging**